view/js/filebrowser.js

```diff
@@ -47,98 +47,115 @@
  *		$("body").on("fbrowser.image.example", function(event, filename, bbcode, id) {
  *			// close colorbox
  *			$.colorbox.close();
- *			// replace textxarea text with bbcode
+ *			// replace textarea text with bbcode
  *			$(id).value = bbcode;
  *		});
  **/
+const FileBrowser = {
+	nickname: '',
+	type: '',
+	event: '',
+	id: null,
 
-var FileBrowser = {
-	nickname : "",
-	type : "",
-	event: "",
-	id : null,
-
-	init: function(nickname, type) {
+	init: function (nickname, type) {
 		FileBrowser.nickname = nickname;
 		FileBrowser.type = type;
-		FileBrowser.event = "fbrowser."+type;
-		if (location['hash']!=="") {
-			var h = location['hash'].replace("#","");
+		FileBrowser.event = "fbrowser." + type;
+		if (location['hash'] !== "") {
+			const h = location['hash'].replace('#', '');
 			FileBrowser.event = FileBrowser.event + "." + h.split("-")[0];
 			FileBrowser.id = h.split("-")[1];
 		}
 
-		console.log("FileBrowser:", nickname, type,FileBrowser.event, FileBrowser.id );
+		console.log('FileBrowser:', nickname, type, FileBrowser.event, FileBrowser.id);
 
-		$(".error a.close").on("click", function(e) {
+		$('.error a.close').on('click', function (e) {
 			e.preventDefault();
-			$(".error").addClass("hidden");
+			$('.error').addClass('hidden');
 		});
 
-		$(".folders a, .path a").on("click", function(e){
+		$('.folders a, .path a').on('click', function (e) {
 			e.preventDefault();
-			location.href = baseurl + "/fbrowser/" + FileBrowser.type + "/" + encodeURIComponent(this.dataset.folder) + "?mode=minimal" + location['hash'];
+			location.href = FileBrowser._getUrl("minimal", location['hash'], this.dataset.folder);
+			location.reload();
 		});
 
-		$(".photo-album-photo-link").on('click', function(e){
+		$(".photo-album-photo-link").on('click', function (e) {
 			e.preventDefault();
 
-			var embed = "";
-			if (FileBrowser.type == "image") {
-				embed = "[url="+this.dataset.link+"][img="+this.dataset.img+"]"+this.dataset.alt+"[/img][/url]";
+			let embed = '';
+			if (FileBrowser.type === "photos") {
+				embed = '[url=' + this.dataset.link + '][img=' + this.dataset.img + ']' + this.dataset.alt + '[/img][/url]';
 			}
-			if (FileBrowser.type=="file") {
-				// attachment links are "baseurl/attach/id"; we need id
-				embed = "[attachment]"+this.dataset.link.split("/").pop()+"[/attachment]";
+			if (FileBrowser.type === "attachment") {
+				embed = '[attachment]' + this.dataset.link + '[/attachment]';
 			}
 			console.log(FileBrowser.event, this.dataset.filename, embed, FileBrowser.id);
-			parent.$("body").trigger(FileBrowser.event, [
+			parent.$('body').trigger(FileBrowser.event, [
 				this.dataset.filename,
 				embed,
 				FileBrowser.id
 			]);
 
 		});
 
-		if ($("#upload-image").length)
-			var image_uploader = new window.AjaxUpload(
-				'upload-image',
-				{ action: 'profile/' + FileBrowser.nickname + '/photos/upload?response=json',
+		if ($('#upload-photos').length)
+		{
```

Comment:
This indenting is new, and it doesn't match PSR-2. Granted, this is Javascript so it doesn't apply, could you explain your intent?

Reply:
I reverted it, it was a mistake from my IDE